src/MSBuild/app.config

```diff
@@ -41,11 +41,64 @@
           <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="15.1.0.0" />
         </dependentAssembly>
 
+        <!-- Redirects for assemblies redistributed by MSBuild (in the .vsix). -->
```

Comment:
We need these in `app.amd64.config` too, right? Also with codebases?

Reply:
Oh yeah, completely forgot about that :)

Reply:
Turns out the vsix duplicates the redistributed dlls for both 32 and 64 dirs, so no codebases required.